tensorflow_probability/python/stats/sample_stats.py

```diff
@@ -645,6 +807,104 @@ def variance(x, sample_axis=0, keepdims=False, name=None):
         x, y=None, sample_axis=sample_axis, event_axis=None, keepdims=keepdims)
 
 
+VarianceState = collections.namedtuple('VarianceState',
+                                       'num_samples, mean, m2, dtype')
+
+
+def init_running_variance(shape=(), dtype=tf.float32):
+  """Initialize the Running Variance State.
+
+  Args:
+    shape: Python `Tuple` or `TensorShape` representing the shape of
+      incoming samples. Scalar samples are assumed by default.
+    dtype: Dtype of incoming samples and the resulting statistics.
+      By default, the dtype is `tf.float32`
+
+  Returns:
+    state: `VarianceState` representing a stream of no inputs
+  """
+  return VarianceState(
+      num_samples=tf.zeros((), dtype=dtype),
+      mean=tf.zeros(shape, dtype=dtype),
+      m2=tf.zeros(shape, dtype=dtype),
+      dtype=dtype,
+  )
+
+
+def update_running_variance(curr_state, new_sample, axis=None):
+  """Update the streaming statistics of the `VarianceState` with a new sample
```

Comment:
Keeping wording consistent:
```suggestion
  """Update the running statistics of the `VarianceState` with a new sample
```